src/components/Resume/ResumeCreation.jsx

```diff
@@ -0,0 +1,225 @@
+import * as Yup from "yup";
+import companylogo from "../../assets/company-logo.png";
+import { postResume } from "./ResumeCreationApi.js";
+import { useFormik } from "formik";
+import { Button, TextField } from "@material-ui/core";
+import { Link, useHistory } from "react-router-dom";
+import "./ResumeCreation.css";
+
+const validationSchema = Yup.object({
+  birthdate: Yup.string()
+    .matches(
+      /^([0-2][0-9]|(3)[0-1])(\/)(((0)[0-9])|((1)[0-2]))(\/)\d{4}$/,
+      "Please enter valid date of birth"
+    )
+    .required("Date of birth field is required"),
+  direction: Yup.string()
+    .min(3, "Address is too short - should be 3 characters minimum")
+    .max(20, "Please enter an address with 20 characters or less")
+    .required("Address field is required"),
+  email: Yup.string()
+    .email("Invalid email address")
+    .required("Email field is required"),
+  firstName: Yup.string()
+    .min(3, "Name is too short - should be 3 characters minimum")
+    .matches(/^[A-Za-z ]*$/, "Please enter valid name")
+    .max(20, "Please enter a name with 20 characters or less")
+    .required("Name field is required"),
+  lastName: Yup.string()
+    .min(5, "Lastname is too short - should be 5 characters minimum")
```

Comment:
Maybe a minimum of 5 is too restrictive, there are lastnames as "Lara" or "Cruz" or "Oña" with less than 5 characters

Reply:
Ok, noted 👍🏻

Reply:
Good catch @paulo-teran-jalafund